packages/cyberstorm/src/components/Dialog/Dialog.module.css

```diff
@@ -31,8 +31,26 @@
   overflow: auto;
 }
 
-.bodyPadding {
-  padding: var(--space--16) var(--space--32) var(--space--32) var(--space--32);
+.bodyPadding > * {
+  padding-right: var(--space--32);
+  padding-left: var(--space--32);
+}
+
+.bodyPadding > :first-child {
+  padding-top: var(--space--16);
+}
+
+.bodyPadding > :last-child {
+  padding-bottom: var(--space--16);
+}
+
+.bodyPadding:where(.footerBorder) > :nth-child(n - 2) {
```

Comment:
What is `n - 2` supposed to select, and why do we want to use it here when we don't know what the content of the children will be? I first assumed it would apply to all but first three (or two) children, but testing on a [online test bench](https://nth-child.belter.io/) seems to show it applies to all children. Is this a bug?

Reply:
Second last, but seems like it worked by accident on my env. Fixed it now